src/identity.ts

```diff
@@ -1,7 +1,7 @@
 import { cryptoWaitReady } from "@polkadot/util-crypto";
 import { Keyring } from "@polkadot/api";
 import { KeyringPair } from "@polkadot/keyring/types";
-import { appSettings, accountSettings, keyringOptions } from "./config";
+import { appSettings, accountSettings, keyringOptions } from "./config/index.js";
```

Comment:
Having to use `.js` extension in `.ts` file is super weird, why is it necessary?

Reply:
Using `ESM` require to set a full relative path for files, including the extension, after reading the link I'am getting why you don't like ESM. 
Back to `commonjs` will remove this.